templates/admin_transfers_section.php

```diff
@@ -97,6 +102,14 @@
     for($o=0; $o<$total_count; $o+=$page_size) {
```

Comment:
total_count was limited before. If this is a large installation it might become quite large and so the for loop will be over many, many tuples. I have local update soon to be a PR to work around this by starting and ending at more specific indexes so that only `$page_size * $display_page_num` before/after should be considered in the loop.